### ISSUE TITLE
Disable front end banner by props 

### ISSUE THREAD

**Issue by Arashfa0301** [webkom/lego#3528]
Currently we comment out the banner each time we want to disable it. This should be changed to be disabled by a prop instead. Or better yet implement accessibility by an admin dynamically
